libobs/obs-source.c

```diff
@@ -391,6 +391,17 @@ static void duplicate_filters(obs_source_t *dst, obs_source_t *src,
 	da_free(filters);
 }
 
+void copy_paste_filters(obs_source_t *dst, obs_source_t *src)
+{
+	bool create_private;
+
+	duplicate_filters(dst, src, create_private ?
```

Comment:
Careful here, variable was used while uninitialized.  This should have been caught by your compiler.